### DISCUSSION TITLE
I want to custom table nested page navigation.

### DISCUSSION THREAD

**Discussion by calakersky** [filamentphp/filament#12062] [Help]
### Package

Table builder

### Package Version

v3

### How can we help you?

I want to custom table like the video below, How to do kindly help. I have the parent users and the child users. When click on the parent list the child of users.

[sm.webm](https://github.com/filamentphp/filament/assets/66521798/50409614-2129-4c6a-848b-859611e48148)

**Comment by micraux** [comment]
You can use a [relation manager](https://filamentphp.com/docs/3.x/panels/resources/relation-managers)
